tests/test65.c

```diff
@@ -16,19 +16,6 @@ struct CacheEntry {
 };
 struct CacheEntry *cache = NULL;
 
-static char * /*value*/ find_in_cache(const char *key)
```

Comment:
I thought so too, that's why I left it in as a comment in case the later unit test should be extended. By only removing the static keyword you will not only have dead code in the .c file, but also in the resultant binary. Given your penchant for small diffs on the autogenerated docs, I'm a bit surprised you'd want to go this route ;)